packages/suite-data/src/message-system/config/config.v1.json

```diff
@@ -1,6 +1,39 @@
 {
     "version": 1,
-    "timestamp": "2021-07-131T11:20:00+00:00",
-    "sequence": 3,
-    "actions": []
+    "timestamp": "2021-09-16T00:00:00+00:00",
+    "sequence": 4,
+    "actions": [
+        {
+            "conditions": [
+                {
+                    "environment": {
+                        "desktop": "21.9.1",
+                        "mobile": "!",
+                        "web": "!"
+                    }
+                }
+            ],
+            "message": {
+                "id": "40d88610-170f-11ec-9621-0242ac130002",
+                "priority": 91,
+                "dismissible": true,
+                "variant": "critical",
+                "category": "banner",
+                "content": {
+                    "en-GB": "There has been an error updating Trezor Suite. Please install the latest version manually from the Trezor Suite homepage.",
```

Comment:
Do we have to have these twice? I would have thought that "en" matches "en-*" (that is it matches "en-GB" as well), or is that not the case?

Reply:
Unfortunately yes. In versions without Spanish lang, there was hardcoded `en-GB` in the message system. Now it correctly uses language property, which is `en` and `es`.